client/components/marketing-survey/gsuite-cancel-purchase-dialog/gsuite-cancellation-features.jsx

```diff
@@ -26,8 +25,7 @@ class GSuiteCancellationFeatures extends Component {
 
 	render() {
 		const { purchase, translate } = this.props;
-		const gsuiteDomain = purchaseType( purchase );
-		const { productSlug } = purchase;
+		const { meta: gsuiteDomain, productSlug } = purchase;
```

Comment:
I really don't know why we used `purchaseType()` to retrieve the domain name 🤦‍♂️.